### ISSUE TITLE
OWASP 5.2.4 cli gives incorrect severity for various node modules

### ISSUE THREAD

**Issue by Anshu2405** [dependency-check/DependencyCheck#2392]
**Describe the bug**
OWASP 5.2.4 cli gives incorrect severity i.e. 0.0 for various node modules when the source of vulnerability is OSSIndex.

**Version**
5.2.4

**Output Logs:**
      "packages": [
        {
          "id": "pkg:npm\/angular@1.3.19",
          "confidence": "HIGHEST",
          "url": "https:\/\/ossindex.sonatype.org\/component\/pkg:npm\/angular@1.3.19"
        }
      ],
        {
          "source": "OSSINDEX",
          "name": "Cross Site Scripting (XSS) in JSONP",
          "severity": "0.0",
          "cwes": [
          ],
          "description": "JSONP allows untrusted resource URLs, which provides a vector for attack by malicious actors.",
          "notes": "",
          "references": [
            {
              "source": "OSSINDEX",
              "url": "https:\/\/ossindex.sonatype.org\/vuln\/9635e385-cf30-49bd-8f4a-1b6c9757d8a3",
              "name": "Cross Site Scripting (XSS) in JSONP"
            }
          ],
          "vulnerableSoftware": [
            {
              "software": {
                "id": "cpe:2.3:a:*:angular:1.3.19:*:*:*:*:*:*:*",
                "vulnerabilityIdMatched": "true"
              }
            }
          ]

**To Reproduce**
Add below dependency and run dependency-check cli using v5.2.4
"angular":"1.3.19"

**Expected behavior**
Severity should not be assigned as 0.0 .Also, what is the severity level for this kind of vulnerability.

**Comment by mprins** [comment]
looking at https://ossindex.sonatype.org/vuln/9635e385-cf30-49bd-8f4a-1b6c9757d8a3 there is no severity score, so if a bug at all, it's a bug in the data

**Comment by Anshu2405** [comment]
Yes , but the results from OSS Index API is with CVSS Score 0. So, it might be a bug in the tool where severity is assigned as 0.0 score instead of CVSS Score.
I have one more doubt regarding the severity level  for such kind of vulnerabilities. Can we consider  it to be a "LOW" severity vulnerability ? 

"vulnerabilities": [
    {
      "id": "9635e385-cf30-49bd-8f4a-1b6c9757d8a3",
      "title": "Cross Site Scripting (XSS) in JSONP",
      "description": "JSONP allows untrusted resource URLs, which provides a vector for attack by malicious actors.",
      "cvssScore": 0,
      "reference": "https://ossindex.sonatype.org/vuln/9635e385-cf30-49bd-8f4a-1b6c9757d8a3"
    },

**Comment by jeremylong** [comment]
I believe this is a duplicate of (or possibly a continuation of) #1605.  The report is being updated in 5.2.5 (being released in the next few weeks) to correctly label these as unscored.  The HTML report will show:

```
Unscored:
     Severity: Unkown
```

After reviewing this ticket I realized what was done for #1605 does not completely cover the XML and JSON formats.  As such, we reved the XML schema version to include `<severity unscored="true">Unknown</severity>` and for the JSON you will see:

```json
                    "unscored": "true",
                    "severity": "Unknown",
```

The reason for "unknown" is that we truly do not know the severity.  I could see some treating these as low, some treating these as critical, and others doing their own evaluation on unscored/unknown vulnerabilities.
